### ISSUE TITLE
anchor tags have no link content or id, buttons would be better

### ISSUE THREAD

**Issue by iainsimmons** [RIP21/react-simplemde-editor#42]
For accessibility reasons, anchor tags should have valid link content/text or at the very least, an `id` attribute.

I think an `id` would maybe be the easiest to add, maybe with the prefix from the instance of the editor itself so you can ensure a unique value if multiple editors are on a page.

Ideally, you would replace them all with `button` elements, which would probably be more semantically correct. See the [recommendation regarding accessibility in the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Accessibility_recommendations)

**Comment by jrsinclair** [comment]
@iainsimmons I think this issue is more to do with [SimpleMDE Editor](https://github.com/NextStepWebs/simplemde-markdown-editor) rather than this one. Might be better to raise the issue there. This module is just providing a React wrapper around SimpleMDE.

**Comment by jrsinclair** [comment]
I've created a PR for this issue at https://github.com/NextStepWebs/simplemde-markdown-editor/pull/523

If that is merged, then in theory, we should just be able to update the dependency.

**Comment by RIP21** [comment]
@iainsimmons @jrsinclair hi guys. I took over this project. If you have any PRs to apply please do! Thanks.

**Comment by jrsinclair** [comment]
Hi @RIP21. PR for this project is at #43

**Comment by RIP21** [comment]
Merged! #43
